random-nugg.html

```diff
@@ -38,10 +39,10 @@
             })
             .css({
                 position:"absolute",
-                left:e.pageX - 150,
-                top:e.pageY - 150,
-                width: '300px',
-                height: '300px',
+                left:e.pageX - (randSize/2),
```

Comment:
are these changes not related to the font styling changes? (I assume yes) if so, please submit this as a unique PR